playbooks/terraform-provider-opentelekomcloud-acceptance-test-opentelekomcloud/run.yaml

```diff
@@ -40,12 +39,19 @@
               export OS_NETWORK_ID="$(openstack network create $OS_NETWORK_NAME -f value -c id)"
               openstack subnet create --network $OS_NETWORK_ID --subnet-range 10.0.20.0/24 $_NET_PREFIX-subnet
           fi
-
+          export OS_VPC_ID="$(openstack router show $_NET_PREFIX-vpc -f value -c id)"
+          if [ -z "$OS_VPC_ID" ]; then
+              export OS_VPC_ID="$(openstack router create $_NET_PREFIX-vpc -f value -c id)"
+              openstack router add subnet $_NET_PREFIX-vpc $_NET_PREFIX-subnet
+          fi
           export OS_SWIFT_ENVIRONMENT=1
+
           # Run except the DNS/FW/LB test 100 testcases at a time
-          testcases=`go test github.com/gator1/terraform-provider-opentelekomcloud/opentelekomcloud/ -v -list 'Acc'`
+          testcases=`go test ./opentelekomcloud/ -v -list 'Acc'`
           testcases=`echo "$testcases" | sed '$d' | grep -v -e FW -e LB`
-          # echo "$testcases" | xargs -t -n100 sh -c 'TF_ACC=1 go test github.com/gator1/terraform-provider-opentelekomcloud/opentelekomcloud/ -v -timeout 120m -run $(echo "$@" | tr " " "|")' argv0 2>&1 | tee $TEST_RESULTS_TXT
+          echo "$testcases" | xargs -t -n100 sh -c 'TF_ACC=1 go test ./opentelekomcloud/ -v -timeout 120m -run $(echo "$@" | tr " " "|")' argv0 2>&1 | tee $TEST_RESULTS_TXT
+          # TF_ACC=1 go test ./opentelekomcloud/ -v -run TestAccNetworkingV2Subnet_basic 2>&1 | tee $TEST_RESULTS_TXT
```

Comment:
There is no need to keep this line in code.

Reply:
yes, will clean it